src/book/02-system/04-hardware/01-overview.mdx

```diff
@@ -18,16 +18,20 @@ The NUbots team uses the NUgus hardware platform, a modification of the [igus®
 
 The following modifications were made to fit our specific requirements:
 
-- The components of the robot were split to to fit the size of the printer bed and volume.
-- The torso was modified to have open shoulder covers for shoulder servo cable access.
+- Split components of the robot to fit the size of the printer bed and volume.
+- Modified the torso to have open shoulder covers for shoulder servo cable access.
+- Redesigned the legs with a simpler model made out of aluminium, with XH540-W270-R servos rather than MX-106 servos, and onyx connectors for the hip and ankle blocks and onyx supports inside the legs.
```

Comment:
```suggestion
- Redesigned the legs with a simpler model made out of aluminium, with XH540-W270-R servos rather than MX-106 servos, onyx connectors for the hip and ankle blocks and onyx supports inside the legs.
```